src/PackagedTests/ComInterfaceUnitTest.cs

```diff
@@ -0,0 +1,40 @@
+﻿
+using System;
+using System.Collections.Generic;
+using Microsoft.Management.Deployment;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace PackagedUnitTests
+{
+    [TestClass]
+    public class ComInterfaceUnitTests
+    {
+        [TestMethod]
+        public void OpenPredefinedCatalog()

```

Comment:
Hopefully a few more tests could be added leveraging the existing local source infrastructure to exercise the API more extensively.